org/addReviewer.ts

```diff
@@ -0,0 +1,39 @@
+import { danger, peril, GitHubUser } from "danger"
+import { PRReviewMetadata } from "../tasks/prReviewReminder"
+
+// Remind reviewers if a review hasn't been received in 1 business day. Does not support teams, only individual users
+// https://github.com/artsy/README/issues/177
+export const rfc177_2 = () => {
+  const now = new Date()
+  if (danger.github.requested_reviewers.users && danger.github.requested_reviewers.users.length > 0) {
+    scheduleReviewReminders(now)
+  }
+}
+
+export const scheduleReviewReminders = (now: Date) => {
+  // Get the day of the week & make it more human-readable
+  const days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+  const day = days[now.getDay()]
+
+  const pr = danger.github.pr
+
+  const metadata: PRReviewMetadata = {
+    repoName: pr.base.repo.name,
+    prNumber: pr.number,
+    reviewer: "me", // TODO: this should be the requested_reviewer associated with the pull_request.review_requested event
```

Comment:
@ashfurrow this is the main bit that I'm struggling with - I could just be missing the right way to reference the `requested_reviewer` field on the event, or it might not be supported by danger.

For reference, this is a full JSON example of one of those events:
```
{
  "action": "review_requested",
  "number": 117,
  "pull_request": {
    "url": "https://api.github.com/repos/artsy/peril-settings/pulls/117",
    "id": 275839123,
    "node_id": "MDExOlB1bGxSZXF1ZXN0Mjc1ODM5MTIz",
    "html_url": "https://github.com/artsy/peril-settings/pull/117",
    "diff_url": "https://github.com/artsy/peril-settings/pull/117.diff",
    "patch_url": "https://github.com/artsy/peril-settings/pull/117.patch",
    "issue_url": "https://api.github.com/repos/artsy/peril-settings/issues/117",
    "number": 117,
    "state": "open",
    "locked": false,
    "title": "[please ignore] testing webhooks",
    "user": {
      "login": "mdole",
      "id": 5361806,
      "node_id": "MDQ6VXNlcjUzNjE4MDY=",
      "avatar_url": "https://avatars1.githubusercontent.com/u/5361806?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/mdole",
      "html_url": "https://github.com/mdole",
      "followers_url": "https://api.github.com/users/mdole/followers",
      "following_url": "https://api.github.com/users/mdole/following{/other_user}",
      "gists_url": "https://api.github.com/users/mdole/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/mdole/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/mdole/subscriptions",
      "organizations_url": "https://api.github.com/users/mdole/orgs",
      "repos_url": "https://api.github.com/users/mdole/repos",
      "events_url": "https://api.github.com/users/mdole/events{/privacy}",
      "received_events_url": "https://api.github.com/users/mdole/received_events",
      "type": "User",
      "site_admin": false
    },
    "body": "@ashfurrow no need to actually review here, I'm just trying to test + record the payload from a `pull_request.review_requested` event using [peril](https://staging-dashboard.peril.systems/installation/185263/webhooks)\r\n\r\n(same to you @dblandin thanks for being my test cases 😄 )",
    "created_at": "2019-05-03T20:26:42Z",
    "updated_at": "2019-05-03T20:29:22Z",
    "closed_at": null,
    "merged_at": null,
    "merge_commit_sha": "91289aecb6500d358f8e6069989e0d0b9204198a",
    "assignee": {
      "login": "mdole",
      "id": 5361806,
      "node_id": "MDQ6VXNlcjUzNjE4MDY=",
      "avatar_url": "https://avatars1.githubusercontent.com/u/5361806?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/mdole",
      "html_url": "https://github.com/mdole",
      "followers_url": "https://api.github.com/users/mdole/followers",
      "following_url": "https://api.github.com/users/mdole/following{/other_user}",
      "gists_url": "https://api.github.com/users/mdole/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/mdole/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/mdole/subscriptions",
      "organizations_url": "https://api.github.com/users/mdole/orgs",
      "repos_url": "https://api.github.com/users/mdole/repos",
      "events_url": "https://api.github.com/users/mdole/events{/privacy}",
      "received_events_url": "https://api.github.com/users/mdole/received_events",
      "type": "User",
      "site_admin": false
    },
    "assignees": [
      {
        "login": "mdole",
        "id": 5361806,
        "node_id": "MDQ6VXNlcjUzNjE4MDY=",
        "avatar_url": "https://avatars1.githubusercontent.com/u/5361806?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/mdole",
        "html_url": "https://github.com/mdole",
        "followers_url": "https://api.github.com/users/mdole/followers",
        "following_url": "https://api.github.com/users/mdole/following{/other_user}",
        "gists_url": "https://api.github.com/users/mdole/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/mdole/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/mdole/subscriptions",
        "organizations_url": "https://api.github.com/users/mdole/orgs",
        "repos_url": "https://api.github.com/users/mdole/repos",
        "events_url": "https://api.github.com/users/mdole/events{/privacy}",
        "received_events_url": "https://api.github.com/users/mdole/received_events",
        "type": "User",
        "site_admin": false
      }
    ],
    "requested_reviewers": [
      {
        "login": "dblandin",
        "id": 123595,
        "node_id": "MDQ6VXNlcjEyMzU5NQ==",
        "avatar_url": "https://avatars3.githubusercontent.com/u/123595?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/dblandin",
        "html_url": "https://github.com/dblandin",
        "followers_url": "https://api.github.com/users/dblandin/followers",
        "following_url": "https://api.github.com/users/dblandin/following{/other_user}",
        "gists_url": "https://api.github.com/users/dblandin/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/dblandin/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/dblandin/subscriptions",
        "organizations_url": "https://api.github.com/users/dblandin/orgs",
        "repos_url": "https://api.github.com/users/dblandin/repos",
        "events_url": "https://api.github.com/users/dblandin/events{/privacy}",
        "received_events_url": "https://api.github.com/users/dblandin/received_events",
        "type": "User",
        "site_admin": false
      },
      {
        "login": "ashfurrow",
        "id": 498212,
        "node_id": "MDQ6VXNlcjQ5ODIxMg==",
        "avatar_url": "https://avatars3.githubusercontent.com/u/498212?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/ashfurrow",
        "html_url": "https://github.com/ashfurrow",
        "followers_url": "https://api.github.com/users/ashfurrow/followers",
        "following_url": "https://api.github.com/users/ashfurrow/following{/other_user}",
        "gists_url": "https://api.github.com/users/ashfurrow/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/ashfurrow/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/ashfurrow/subscriptions",
        "organizations_url": "https://api.github.com/users/ashfurrow/orgs",
        "repos_url": "https://api.github.com/users/ashfurrow/repos",
        "events_url": "https://api.github.com/users/ashfurrow/events{/privacy}",
        "received_events_url": "https://api.github.com/users/ashfurrow/received_events",
        "type": "User",
        "site_admin": false
      }
    ],
    "requested_teams": [],
    "labels": [],
    "milestone": null,
    "commits_url": "https://api.github.com/repos/artsy/peril-settings/pulls/117/commits",
    "review_comments_url": "https://api.github.com/repos/artsy/peril-settings/pulls/117/comments",
    "review_comment_url": "https://api.github.com/repos/artsy/peril-settings/pulls/comments{/number}",
    "comments_url": "https://api.github.com/repos/artsy/peril-settings/issues/117/comments",
    "statuses_url": "https://api.github.com/repos/artsy/peril-settings/statuses/f3ad76c696824fcd9c80ea62497de48fe25e77ad",
    "head": {
      "label": "mdole:testing-webhooks",
      "ref": "testing-webhooks",
      "sha": "f3ad76c696824fcd9c80ea62497de48fe25e77ad",
      "user": {
        "login": "mdole",
        "id": 5361806,
        "node_id": "MDQ6VXNlcjUzNjE4MDY=",
        "avatar_url": "https://avatars1.githubusercontent.com/u/5361806?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/mdole",
        "html_url": "https://github.com/mdole",
        "followers_url": "https://api.github.com/users/mdole/followers",
        "following_url": "https://api.github.com/users/mdole/following{/other_user}",
        "gists_url": "https://api.github.com/users/mdole/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/mdole/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/mdole/subscriptions",
        "organizations_url": "https://api.github.com/users/mdole/orgs",
        "repos_url": "https://api.github.com/users/mdole/repos",
        "events_url": "https://api.github.com/users/mdole/events{/privacy}",
        "received_events_url": "https://api.github.com/users/mdole/received_events",
        "type": "User",
        "site_admin": false
      },
      "repo": {
        "id": 182870231,
        "node_id": "MDEwOlJlcG9zaXRvcnkxODI4NzAyMzE=",
        "name": "peril-settings",
        "full_name": "mdole/peril-settings",
        "private": false,
        "owner": {
          "login": "mdole",
          "id": 5361806,
          "node_id": "MDQ6VXNlcjUzNjE4MDY=",
          "avatar_url": "https://avatars1.githubusercontent.com/u/5361806?v=4",
          "gravatar_id": "",
          "url": "https://api.github.com/users/mdole",
          "html_url": "https://github.com/mdole",
          "followers_url": "https://api.github.com/users/mdole/followers",
          "following_url": "https://api.github.com/users/mdole/following{/other_user}",
          "gists_url": "https://api.github.com/users/mdole/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/mdole/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/mdole/subscriptions",
          "organizations_url": "https://api.github.com/users/mdole/orgs",
          "repos_url": "https://api.github.com/users/mdole/repos",
          "events_url": "https://api.github.com/users/mdole/events{/privacy}",
          "received_events_url": "https://api.github.com/users/mdole/received_events",
          "type": "User",
          "site_admin": false
        },
        "html_url": "https://github.com/mdole/peril-settings",
        "description": "Artsy's peril settings",
        "fork": true,
        "url": "https://api.github.com/repos/mdole/peril-settings",
        "forks_url": "https://api.github.com/repos/mdole/peril-settings/forks",
        "keys_url": "https://api.github.com/repos/mdole/peril-settings/keys{/key_id}",
        "collaborators_url": "https://api.github.com/repos/mdole/peril-settings/collaborators{/collaborator}",
        "teams_url": "https://api.github.com/repos/mdole/peril-settings/teams",
        "hooks_url": "https://api.github.com/repos/mdole/peril-settings/hooks",
        "issue_events_url": "https://api.github.com/repos/mdole/peril-settings/issues/events{/number}",
        "events_url": "https://api.github.com/repos/mdole/peril-settings/events",
        "assignees_url": "https://api.github.com/repos/mdole/peril-settings/assignees{/user}",
        "branches_url": "https://api.github.com/repos/mdole/peril-settings/branches{/branch}",
        "tags_url": "https://api.github.com/repos/mdole/peril-settings/tags",
        "blobs_url": "https://api.github.com/repos/mdole/peril-settings/git/blobs{/sha}",
        "git_tags_url": "https://api.github.com/repos/mdole/peril-settings/git/tags{/sha}",
        "git_refs_url": "https://api.github.com/repos/mdole/peril-settings/git/refs{/sha}",
        "trees_url": "https://api.github.com/repos/mdole/peril-settings/git/trees{/sha}",
        "statuses_url": "https://api.github.com/repos/mdole/peril-settings/statuses/{sha}",
        "languages_url": "https://api.github.com/repos/mdole/peril-settings/languages",
        "stargazers_url": "https://api.github.com/repos/mdole/peril-settings/stargazers",
        "contributors_url": "https://api.github.com/repos/mdole/peril-settings/contributors",
        "subscribers_url": "https://api.github.com/repos/mdole/peril-settings/subscribers",
        "subscription_url": "https://api.github.com/repos/mdole/peril-settings/subscription",
        "commits_url": "https://api.github.com/repos/mdole/peril-settings/commits{/sha}",
        "git_commits_url": "https://api.github.com/repos/mdole/peril-settings/git/commits{/sha}",
        "comments_url": "https://api.github.com/repos/mdole/peril-settings/comments{/number}",
        "issue_comment_url": "https://api.github.com/repos/mdole/peril-settings/issues/comments{/number}",
        "contents_url": "https://api.github.com/repos/mdole/peril-settings/contents/{+path}",
        "compare_url": "https://api.github.com/repos/mdole/peril-settings/compare/{base}...{head}",
        "merges_url": "https://api.github.com/repos/mdole/peril-settings/merges",
        "archive_url": "https://api.github.com/repos/mdole/peril-settings/{archive_format}{/ref}",
        "downloads_url": "https://api.github.com/repos/mdole/peril-settings/downloads",
        "issues_url": "https://api.github.com/repos/mdole/peril-settings/issues{/number}",
        "pulls_url": "https://api.github.com/repos/mdole/peril-settings/pulls{/number}",
        "milestones_url": "https://api.github.com/repos/mdole/peril-settings/milestones{/number}",
        "notifications_url": "https://api.github.com/repos/mdole/peril-settings/notifications{?since,all,participating}",
        "labels_url": "https://api.github.com/repos/mdole/peril-settings/labels{/name}",
        "releases_url": "https://api.github.com/repos/mdole/peril-settings/releases{/id}",
        "deployments_url": "https://api.github.com/repos/mdole/peril-settings/deployments",
        "created_at": "2019-04-22T21:22:28Z",
        "updated_at": "2019-04-29T16:59:58Z",
        "pushed_at": "2019-05-03T20:24:09Z",
        "git_url": "git://github.com/mdole/peril-settings.git",
        "ssh_url": "git@github.com:mdole/peril-settings.git",
        "clone_url": "https://github.com/mdole/peril-settings.git",
        "svn_url": "https://github.com/mdole/peril-settings",
        "homepage": "",
        "size": 816,
        "stargazers_count": 0,
        "watchers_count": 0,
        "language": "TypeScript",
        "has_issues": false,
        "has_projects": true,
        "has_downloads": true,
        "has_wiki": false,
        "has_pages": false,
        "forks_count": 0,
        "mirror_url": null,
        "archived": false,
        "disabled": false,
        "open_issues_count": 0,
        "license": {
          "key": "mit",
          "name": "MIT License",
          "spdx_id": "MIT",
          "url": "https://api.github.com/licenses/mit",
          "node_id": "MDc6TGljZW5zZTEz"
        },
        "forks": 0,
        "open_issues": 0,
        "watchers": 0,
        "default_branch": "master"
      }
    },
    "base": {
      "label": "artsy:master",
      "ref": "master",
      "sha": "7ca0a759e66d44ea0e629c8d56d73b4e90788934",
      "user": {
        "login": "artsy",
        "id": 546231,
        "node_id": "MDEyOk9yZ2FuaXphdGlvbjU0NjIzMQ==",
        "avatar_url": "https://avatars3.githubusercontent.com/u/546231?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/artsy",
        "html_url": "https://github.com/artsy",
        "followers_url": "https://api.github.com/users/artsy/followers",
        "following_url": "https://api.github.com/users/artsy/following{/other_user}",
        "gists_url": "https://api.github.com/users/artsy/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/artsy/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/artsy/subscriptions",
        "organizations_url": "https://api.github.com/users/artsy/orgs",
        "repos_url": "https://api.github.com/users/artsy/repos",
        "events_url": "https://api.github.com/users/artsy/events{/privacy}",
        "received_events_url": "https://api.github.com/users/artsy/received_events",
        "type": "Organization",
        "site_admin": false
      },
      "repo": {
        "id": 68032702,
        "node_id": "MDEwOlJlcG9zaXRvcnk2ODAzMjcwMg==",
        "name": "peril-settings",
        "full_name": "artsy/peril-settings",
        "private": false,
        "owner": {
          "login": "artsy",
          "id": 546231,
          "node_id": "MDEyOk9yZ2FuaXphdGlvbjU0NjIzMQ==",
          "avatar_url": "https://avatars3.githubusercontent.com/u/546231?v=4",
          "gravatar_id": "",
          "url": "https://api.github.com/users/artsy",
          "html_url": "https://github.com/artsy",
          "followers_url": "https://api.github.com/users/artsy/followers",
          "following_url": "https://api.github.com/users/artsy/following{/other_user}",
          "gists_url": "https://api.github.com/users/artsy/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/artsy/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/artsy/subscriptions",
          "organizations_url": "https://api.github.com/users/artsy/orgs",
          "repos_url": "https://api.github.com/users/artsy/repos",
          "events_url": "https://api.github.com/users/artsy/events{/privacy}",
          "received_events_url": "https://api.github.com/users/artsy/received_events",
          "type": "Organization",
          "site_admin": false
        },
        "html_url": "https://github.com/artsy/peril-settings",
        "description": "Artsy's peril settings",
        "fork": false,
        "url": "https://api.github.com/repos/artsy/peril-settings",
        "forks_url": "https://api.github.com/repos/artsy/peril-settings/forks",
        "keys_url": "https://api.github.com/repos/artsy/peril-settings/keys{/key_id}",
        "collaborators_url": "https://api.github.com/repos/artsy/peril-settings/collaborators{/collaborator}",
        "teams_url": "https://api.github.com/repos/artsy/peril-settings/teams",
        "hooks_url": "https://api.github.com/repos/artsy/peril-settings/hooks",
        "issue_events_url": "https://api.github.com/repos/artsy/peril-settings/issues/events{/number}",
        "events_url": "https://api.github.com/repos/artsy/peril-settings/events",
        "assignees_url": "https://api.github.com/repos/artsy/peril-settings/assignees{/user}",
        "branches_url": "https://api.github.com/repos/artsy/peril-settings/branches{/branch}",
        "tags_url": "https://api.github.com/repos/artsy/peril-settings/tags",
        "blobs_url": "https://api.github.com/repos/artsy/peril-settings/git/blobs{/sha}",
        "git_tags_url": "https://api.github.com/repos/artsy/peril-settings/git/tags{/sha}",
        "git_refs_url": "https://api.github.com/repos/artsy/peril-settings/git/refs{/sha}",
        "trees_url": "https://api.github.com/repos/artsy/peril-settings/git/trees{/sha}",
        "statuses_url": "https://api.github.com/repos/artsy/peril-settings/statuses/{sha}",
        "languages_url": "https://api.github.com/repos/artsy/peril-settings/languages",
        "stargazers_url": "https://api.github.com/repos/artsy/peril-settings/stargazers",
        "contributors_url": "https://api.github.com/repos/artsy/peril-settings/contributors",
        "subscribers_url": "https://api.github.com/repos/artsy/peril-settings/subscribers",
        "subscription_url": "https://api.github.com/repos/artsy/peril-settings/subscription",
        "commits_url": "https://api.github.com/repos/artsy/peril-settings/commits{/sha}",
        "git_commits_url": "https://api.github.com/repos/artsy/peril-settings/git/commits{/sha}",
        "comments_url": "https://api.github.com/repos/artsy/peril-settings/comments{/number}",
        "issue_comment_url": "https://api.github.com/repos/artsy/peril-settings/issues/comments{/number}",
        "contents_url": "https://api.github.com/repos/artsy/peril-settings/contents/{+path}",
        "compare_url": "https://api.github.com/repos/artsy/peril-settings/compare/{base}...{head}",
        "merges_url": "https://api.github.com/repos/artsy/peril-settings/merges",
        "archive_url": "https://api.github.com/repos/artsy/peril-settings/{archive_format}{/ref}",
        "downloads_url": "https://api.github.com/repos/artsy/peril-settings/downloads",
        "issues_url": "https://api.github.com/repos/artsy/peril-settings/issues{/number}",
        "pulls_url": "https://api.github.com/repos/artsy/peril-settings/pulls{/number}",
        "milestones_url": "https://api.github.com/repos/artsy/peril-settings/milestones{/number}",
        "notifications_url": "https://api.github.com/repos/artsy/peril-settings/notifications{?since,all,participating}",
        "labels_url": "https://api.github.com/repos/artsy/peril-settings/labels{/name}",
        "releases_url": "https://api.github.com/repos/artsy/peril-settings/releases{/id}",
        "deployments_url": "https://api.github.com/repos/artsy/peril-settings/deployments",
        "created_at": "2016-09-12T17:43:30Z",
        "updated_at": "2019-05-03T17:46:55Z",
        "pushed_at": "2019-05-03T20:26:42Z",
        "git_url": "git://github.com/artsy/peril-settings.git",
        "ssh_url": "git@github.com:artsy/peril-settings.git",
        "clone_url": "https://github.com/artsy/peril-settings.git",
        "svn_url": "https://github.com/artsy/peril-settings",
        "homepage": "",
        "size": 734,
        "stargazers_count": 16,
        "watchers_count": 16,
        "language": "TypeScript",
        "has_issues": true,
        "has_projects": false,
        "has_downloads": true,
        "has_wiki": false,
        "has_pages": false,
        "forks_count": 15,
        "mirror_url": null,
        "archived": false,
        "disabled": false,
        "open_issues_count": 12,
        "license": {
          "key": "mit",
          "name": "MIT License",
          "spdx_id": "MIT",
          "url": "https://api.github.com/licenses/mit",
          "node_id": "MDc6TGljZW5zZTEz"
        },
        "forks": 15,
        "open_issues": 12,
        "watchers": 16,
        "default_branch": "master"
      }
    },
    "_links": {
      "self": {
        "href": "https://api.github.com/repos/artsy/peril-settings/pulls/117"
      },
      "html": {
        "href": "https://github.com/artsy/peril-settings/pull/117"
      },
      "issue": {
        "href": "https://api.github.com/repos/artsy/peril-settings/issues/117"
      },
      "comments": {
        "href": "https://api.github.com/repos/artsy/peril-settings/issues/117/comments"
      },
      "review_comments": {
        "href": "https://api.github.com/repos/artsy/peril-settings/pulls/117/comments"
      },
      "review_comment": {
        "href": "https://api.github.com/repos/artsy/peril-settings/pulls/comments{/number}"
      },
      "commits": {
        "href": "https://api.github.com/repos/artsy/peril-settings/pulls/117/commits"
      },
      "statuses": {
        "href": "https://api.github.com/repos/artsy/peril-settings/statuses/f3ad76c696824fcd9c80ea62497de48fe25e77ad"
      }
    },
    "author_association": "NONE",
    "draft": false,
    "merged": false,
    "mergeable": true,
    "rebaseable": true,
    "mergeable_state": "clean",
    "merged_by": null,
    "comments": 0,
    "review_comments": 0,
    "maintainer_can_modify": true,
    "commits": 1,
    "additions": 1,
    "deletions": 1,
    "changed_files": 1
  },
  "requested_reviewer": {
    "login": "dblandin",
    "id": 123595,
    "node_id": "MDQ6VXNlcjEyMzU5NQ==",
    "avatar_url": "https://avatars3.githubusercontent.com/u/123595?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/dblandin",
    "html_url": "https://github.com/dblandin",
    "followers_url": "https://api.github.com/users/dblandin/followers",
    "following_url": "https://api.github.com/users/dblandin/following{/other_user}",
    "gists_url": "https://api.github.com/users/dblandin/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/dblandin/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/dblandin/subscriptions",
    "organizations_url": "https://api.github.com/users/dblandin/orgs",
    "repos_url": "https://api.github.com/users/dblandin/repos",
    "events_url": "https://api.github.com/users/dblandin/events{/privacy}",
    "received_events_url": "https://api.github.com/users/dblandin/received_events",
    "type": "User",
    "site_admin": false
  },
  "repository": {
    "id": 68032702,
    "node_id": "MDEwOlJlcG9zaXRvcnk2ODAzMjcwMg==",
    "name": "peril-settings",
    "full_name": "artsy/peril-settings",
    "private": false,
    "owner": {
      "login": "artsy",
      "id": 546231,
      "node_id": "MDEyOk9yZ2FuaXphdGlvbjU0NjIzMQ==",
      "avatar_url": "https://avatars3.githubusercontent.com/u/546231?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/artsy",
      "html_url": "https://github.com/artsy",
      "followers_url": "https://api.github.com/users/artsy/followers",
      "following_url": "https://api.github.com/users/artsy/following{/other_user}",
      "gists_url": "https://api.github.com/users/artsy/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/artsy/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/artsy/subscriptions",
      "organizations_url": "https://api.github.com/users/artsy/orgs",
      "repos_url": "https://api.github.com/users/artsy/repos",
      "events_url": "https://api.github.com/users/artsy/events{/privacy}",
      "received_events_url": "https://api.github.com/users/artsy/received_events",
      "type": "Organization",
      "site_admin": false
    },
    "html_url": "https://github.com/artsy/peril-settings",
    "description": "Artsy's peril settings",
    "fork": false,
    "url": "https://api.github.com/repos/artsy/peril-settings",
    "forks_url": "https://api.github.com/repos/artsy/peril-settings/forks",
    "keys_url": "https://api.github.com/repos/artsy/peril-settings/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/artsy/peril-settings/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/artsy/peril-settings/teams",
    "hooks_url": "https://api.github.com/repos/artsy/peril-settings/hooks",
    "issue_events_url": "https://api.github.com/repos/artsy/peril-settings/issues/events{/number}",
    "events_url": "https://api.github.com/repos/artsy/peril-settings/events",
    "assignees_url": "https://api.github.com/repos/artsy/peril-settings/assignees{/user}",
    "branches_url": "https://api.github.com/repos/artsy/peril-settings/branches{/branch}",
    "tags_url": "https://api.github.com/repos/artsy/peril-settings/tags",
    "blobs_url": "https://api.github.com/repos/artsy/peril-settings/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/artsy/peril-settings/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/artsy/peril-settings/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/artsy/peril-settings/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/artsy/peril-settings/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/artsy/peril-settings/languages",
    "stargazers_url": "https://api.github.com/repos/artsy/peril-settings/stargazers",
    "contributors_url": "https://api.github.com/repos/artsy/peril-settings/contributors",
    "subscribers_url": "https://api.github.com/repos/artsy/peril-settings/subscribers",
    "subscription_url": "https://api.github.com/repos/artsy/peril-settings/subscription",
    "commits_url": "https://api.github.com/repos/artsy/peril-settings/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/artsy/peril-settings/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/artsy/peril-settings/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/artsy/peril-settings/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/artsy/peril-settings/contents/{+path}",
    "compare_url": "https://api.github.com/repos/artsy/peril-settings/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/artsy/peril-settings/merges",
    "archive_url": "https://api.github.com/repos/artsy/peril-settings/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/artsy/peril-settings/downloads",
    "issues_url": "https://api.github.com/repos/artsy/peril-settings/issues{/number}",
    "pulls_url": "https://api.github.com/repos/artsy/peril-settings/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/artsy/peril-settings/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/artsy/peril-settings/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/artsy/peril-settings/labels{/name}",
    "releases_url": "https://api.github.com/repos/artsy/peril-settings/releases{/id}",
    "deployments_url": "https://api.github.com/repos/artsy/peril-settings/deployments",
    "created_at": "2016-09-12T17:43:30Z",
    "updated_at": "2019-05-03T17:46:55Z",
    "pushed_at": "2019-05-03T20:26:42Z",
    "git_url": "git://github.com/artsy/peril-settings.git",
    "ssh_url": "git@github.com:artsy/peril-settings.git",
    "clone_url": "https://github.com/artsy/peril-settings.git",
    "svn_url": "https://github.com/artsy/peril-settings",
    "homepage": "",
    "size": 734,
    "stargazers_count": 16,
    "watchers_count": 16,
    "language": "TypeScript",
    "has_issues": true,
    "has_projects": false,
    "has_downloads": true,
    "has_wiki": false,
    "has_pages": false,
    "forks_count": 15,
    "mirror_url": null,
    "archived": false,
    "disabled": false,
    "open_issues_count": 12,
    "license": {
      "key": "mit",
      "name": "MIT License",
      "spdx_id": "MIT",
      "url": "https://api.github.com/licenses/mit",
      "node_id": "MDc6TGljZW5zZTEz"
    },
    "forks": 15,
    "open_issues": 12,
    "watchers": 16,
    "default_branch": "master"
  },
  "organization": {
    "login": "artsy",
    "id": 546231,
    "node_id": "MDEyOk9yZ2FuaXphdGlvbjU0NjIzMQ==",
    "url": "https://api.github.com/orgs/artsy",
    "repos_url": "https://api.github.com/orgs/artsy/repos",
    "events_url": "https://api.github.com/orgs/artsy/events",
    "hooks_url": "https://api.github.com/orgs/artsy/hooks",
    "issues_url": "https://api.github.com/orgs/artsy/issues",
    "members_url": "https://api.github.com/orgs/artsy/members{/member}",
    "public_members_url": "https://api.github.com/orgs/artsy/public_members{/member}",
    "avatar_url": "https://avatars3.githubusercontent.com/u/546231?v=4",
    "description": "Artsy.net Team"
  },
  "sender": {
    "login": "mdole",
    "id": 5361806,
    "node_id": "MDQ6VXNlcjUzNjE4MDY=",
    "avatar_url": "https://avatars1.githubusercontent.com/u/5361806?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/mdole",
    "html_url": "https://github.com/mdole",
    "followers_url": "https://api.github.com/users/mdole/followers",
    "following_url": "https://api.github.com/users/mdole/following{/other_user}",
    "gists_url": "https://api.github.com/users/mdole/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/mdole/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/mdole/subscriptions",
    "organizations_url": "https://api.github.com/users/mdole/orgs",
    "repos_url": "https://api.github.com/users/mdole/repos",
    "events_url": "https://api.github.com/users/mdole/events{/privacy}",
    "received_events_url": "https://api.github.com/users/mdole/received_events",
    "type": "User",
    "site_admin": false
  },
  "installation": {
    "id": 185263,
    "node_id": "MDIzOkludGVncmF0aW9uSW5zdGFsbGF0aW9uMTg1MjYz"
  }
}}```

Reply:
Cool, yeah agreed they should be there. No worries – we'll probably need to change the `rfc177_2` function. Peril will pass it the event JSON that you've got there (for the event that invokes it). The typings are a bit complicated, but are available here: https://github.com/orta/github-webhook-event-types They are generated from the GitHub API spec, so they should be complete. It'll just be a matter of finding the one we want (the event that GitHub sends when a pull request review is requested). Looking forward to pairing 👍